tools/osv-scanner/cmd/osv-scanner/config.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/BurntSushi/toml"
+	"golang.org/x/exp/slices"
+)
+
+type ConfigManager struct {
+	// Override to replace all other configs
+	overrideConfig *Config
+	// Config to use if no config file is found alongside manifests
+	defaultConfig Config
+	// Cache to store loaded configs
+	configMap map[string]Config
+}
+
+type Config struct {
+	IgnoredVulns []IgnoreLine
+	LoadPath     string
+}
+
+type IgnoreLine struct {
```

Comment:
nit: "Line" is a bit ambiguous -- I had thought it was at first referring to a line number or a single line, when it doesn't. Can we rename it "IgnoreEntry" or something similar?